dokka-runners/dokka-gradle-plugin/src/main/kotlin/DokkaBasePlugin.kt

```diff
@@ -154,8 +159,14 @@ constructor(
             offlineMode.convention(false)
             outputDirectory.convention(dokkaExtension.dokkaPublicationDirectory.dir(formatName))
             moduleOutputDirectory.convention(dokkaExtension.dokkaModuleDirectory.dir(formatName))
-            suppressInheritedMembers.convention(false)
-            suppressObviousFunctions.convention(true)
+            suppressInheritedMembers.convention(
+                @Suppress("DEPRECATION")
+                dokkaExtension.suppressInheritedMembers.orElse(false)
```

Comment:
why `orElse` is needed here?

Reply:
it's not, thanks for spotting it. I've removed it.